From 7138237cdb51fae238067166bd765f900e68eb7d Mon Sep 17 00:00:00 2001
From: Steve Kowalik <steven@wedontsleep.org>
Date: Mon, 17 Jul 2023 13:22:22 +1000
Subject: [PATCH] Add py_partiql_parser to packages list

setuptools traverses the packages list when deciding what to populate
the sdist and wheels with, and as it stands, only the tests are shipped
in them. Include both py_partiql_parser and the _internal module, so
they are shipped as well.
---
 pyproject.toml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pyproject.toml b/pyproject.toml
index 80675cc..5ebda0d 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -16,6 +16,8 @@ dependencies = []
 
 [tool.setuptools]
 packages = [
+    "py_partiql_parser",
+    "py_partiql_parser._internal",
     "tests"
 ]
 
